README.md

```diff
@@ -0,0 +1,34 @@
+## A simple, ready-to-use docker image of `fhirbase`
+
+A docker simple docker image created from the following ingredients:
+- The official `PostgreSQL` image with `plv8` extension installed
+- Latest **released** `fhirbase`
+
+Together, this creates a very specific, yet stable docker image suitable for all docker environments.
+
+### Usage
+
+Simply 
```

Comment:
Take away simply - if someone doesn't have it working, it won't be simple, and they'll be frustrated by your vocabulary